### ISSUE TITLE
getting OpenSSL.SSL.ZeroReturnError in failure set env variable

### ISSUE THREAD

**Issue by bf4** [deis/deis#4300]
I'm trying to set an env var `deis config:set FOO=bar -a production` and am getting  `OpenSSL.SSL.ZeroReturnError` 

```plain
Creating config... Traceback (most recent call last):
  File "<string>", line 2540, in <module>
  File "<string>", line 2536, in main
  File "<string>", line 2469, in _dispatch_cmd
  File "<string>", line 1290, in config_set
  File "<string>", line 1302, in _config_set
  File "<string>", line 450, in _dispatch
  File "/Users/jenkins/workspace/build-deis-cli-installer-darwin/src/github.com/deis/deis/client/build/deis/out00-PYZ.pyz/requests.sessions", line 504, in post
  File "<string>", line 170, in request
  File "/Users/jenkins/workspace/build-deis-cli-installer-darwin/src/github.com/deis/deis/client/build/deis/out00-PYZ.pyz/requests.sessions", line 461, in request
  File "/Users/jenkins/workspace/build-deis-cli-installer-darwin/src/github.com/deis/deis/client/build/deis/out00-PYZ.pyz/requests.sessions", line 610, in send
  File "/Users/jenkins/workspace/build-deis-cli-installer-darwin/src/github.com/deis/deis/client/build/deis/out00-PYZ.pyz/requests.models", line 730, in content
  File "/Users/jenkins/workspace/build-deis-cli-installer-darwin/src/github.com/deis/deis/client/build/deis/out00-PYZ.pyz/requests.models", line 655, in generate
  File "/Users/jenkins/workspace/build-deis-cli-installer-darwin/src/github.com/deis/deis/client/build/deis/out00-PYZ.pyz/requests.packages.urllib3.response", line 256, in stream
  File "/Users/jenkins/workspace/build-deis-cli-installer-darwin/src/github.com/deis/deis/client/build/deis/out00-PYZ.pyz/requests.packages.urllib3.response", line 186, in read
  File "/Users/jenkins/workspace/build-deis-cli-installer-darwin/src/github.com/deis/deis/client/build/deis/out00-PYZ.pyz/httplib", line 567, in read
  File "/Users/jenkins/workspace/build-deis-cli-installer-darwin/src/github.com/deis/deis/client/build/deis/out00-PYZ.pyz/socket", line 380, in read
  File "/Users/jenkins/workspace/build-deis-cli-installer-darwin/src/github.com/deis/deis/client/build/deis/out00-PYZ.pyz/requests.packages.urllib3.contrib.pyopenssl", line 188, in recv
  File "/Users/jenkins/workspace/build-deis-cli-installer-darwin/src/github.com/deis/deis/client/build/deis/out00-PYZ.pyz/OpenSSL.SSL", line 1320, in recv
  File "/Users/jenkins/workspace/build-deis-cli-installer-darwin/src/github.com/deis/deis/client/build/deis/out00-PYZ.pyz/OpenSSL.SSL", line 1167, in _raise_ssl_error
OpenSSL.SSL.ZeroReturnError
```

This is on a new deis 1.9.0 cluster that I already been set up (i.e. the command has worked before on my machine)

I've included some of the fixes since the v1.9.0 tag in the platform build: `git log v1.9.0-v1.9.0-b90c2fbb --oneline` https://www.irccloud.com/pastebin/z6eiyQOI/

I asked in [#deis](ircs://irc.freenode.net:6697/#deis) and mboersma wrote:

> I haven't seen that error either, although there have been a couple similar issues due to something about how the Deis CLI is packaged by PyInstaller.
> If you can run "deis.py" locally or install it with "pip install deis", I doubt you'll see that problem.
> but if you could enter a GitHub issue, maybe someone actually knows of a fix...

I agree to the [DCO](https://github.com/deis/deis/blob/8ccb76044f06f6a0ee3566868334009198375367/DCO)

Output of `ssh "core@$DEISCTL_TUNNEL" /bin/bash -l -c '/opt/bin/deis-debug-logs'` (slightly amended)

```plain
--- VERSIONS ---
CoreOS 723.3.0
Deis v1.9.0
etcd version 0.4.9
fleetd version 0.10.2

--- SYSTEM STATUS ---
-- Logs begin at Wed 2015-08-19 10:34:41 UTC, end at Wed 2015-08-19 16:39:24 UTC. --
Aug 19 16:39:11 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:11 etcdserver: failed to reach the peerURL(http://10.x.x.x:7001) of member klm987 (Get http://10.x.x.x:7001/version: dial tcp 10.x.x.x:7001: no route to host)
Aug 19 16:39:11 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:11 etcdserver: cannot get the version of member klm987 (Get http://10.x.x.x:7001/version: dial tcp 10.x.x.x:7001: no route to host)
Aug 19 16:39:11 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:11 rafthttp: failed to dial klm987 on stream Message (dial tcp 10.x.x.x:7001: no route to host)
Aug 19 16:39:11 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:11 rafthttp: failed to dial klm987 on stream MsgApp v2 (dial tcp 10.x.x.x:7001: no route to host)
Aug 19 16:39:12 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:12 rafthttp: failed to write klm987 on pipeline (dial tcp 10.x.x.x:2380: i/o timeout)
Aug 19 16:39:12 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:12 rafthttp: failed to dial klm987 on stream Message (dial tcp 10.x.x.x:2380: i/o timeout)
Aug 19 16:39:12 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:12 rafthttp: failed to dial klm987 on stream MsgApp v2 (dial tcp 10.x.x.x:2380: i/o timeout)
Aug 19 16:39:13 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:13 rafthttp: failed to write klm987 on pipeline (dial tcp 10.x.x.x:7001: i/o timeout)
Aug 19 16:39:13 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:13 rafthttp: failed to dial klm987 on stream Message (dial tcp 10.x.x.x:7001: i/o timeout)
Aug 19 16:39:13 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:13 rafthttp: failed to dial klm987 on stream MsgApp v2 (dial tcp 10.x.x.x:7001: i/o timeout)
Aug 19 16:39:14 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:14 rafthttp: failed to dial klm987 on stream MsgApp v2 (dial tcp 10.x.x.x:2380: no route to host)
Aug 19 16:39:14 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:14 rafthttp: failed to dial klm987 on stream Message (dial tcp 10.x.x.x:2380: no route to host)
Aug 19 16:39:14 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:14 rafthttp: failed to write klm987 on pipeline (dial tcp 10.x.x.x:2380: no route to host)
Aug 19 16:39:15 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:15 rafthttp: failed to write klm987 on pipeline (dial tcp 10.x.x.x:7001: i/o timeout)
Aug 19 16:39:15 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:15 rafthttp: failed to dial klm987 on stream MsgApp v2 (dial tcp 10.x.x.x:7001: i/o timeout)
Aug 19 16:39:15 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:15 rafthttp: failed to dial klm987 on stream Message (dial tcp 10.x.x.x:7001: i/o timeout)
Aug 19 16:39:16 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:16 rafthttp: failed to write klm987 on pipeline (dial tcp 10.x.x.x:2380: i/o timeout)
Aug 19 16:39:16 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:16 rafthttp: failed to dial klm987 on stream MsgApp v2 (dial tcp 10.x.x.x:2380: i/o timeout)
Aug 19 16:39:16 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:16 rafthttp: failed to dial klm987 on stream Message (dial tcp 10.x.x.x:2380: i/o timeout)
Aug 19 16:39:17 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:17 etcdserver: failed to reach the peerURL(http://10.x.x.x:2380) of member klm987 (Get http://10.x.x.x:2380/version: net/http: request canceled while waiting for connection)
Aug 19 16:39:17 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:17 etcdserver: failed to reach the peerURL(http://10.x.x.x:7001) of member klm987 (Get http://10.x.x.x:7001/version: dial tcp 10.x.x.x:7001: no route to host)
Aug 19 16:39:17 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:17 etcdserver: cannot get the version of member klm987 (Get http://10.x.x.x:7001/version: dial tcp 10.x.x.x:7001: no route to host)
Aug 19 16:39:17 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:17 rafthttp: failed to dial klm987 on stream Message (dial tcp 10.x.x.x:7001: no route to host)
Aug 19 16:39:17 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:17 rafthttp: failed to dial klm987 on stream MsgApp v2 (dial tcp 10.x.x.x:7001: no route to host)
Aug 19 16:39:17 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:17 rafthttp: failed to write klm987 on pipeline (dial tcp 10.x.x.x:7001: no route to host)
Aug 19 16:39:18 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:18 rafthttp: failed to write klm987 on pipeline (dial tcp 10.x.x.x:2380: i/o timeout)
Aug 19 16:39:18 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:18 rafthttp: failed to dial klm987 on stream Message (dial tcp 10.x.x.x:2380: i/o timeout)
Aug 19 16:39:18 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:18 rafthttp: failed to dial klm987 on stream MsgApp v2 (dial tcp 10.x.x.x:2380: i/o timeout)
Aug 19 16:39:19 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:19 rafthttp: failed to write klm987 on pipeline (dial tcp 10.x.x.x:7001: i/o timeout)
Aug 19 16:39:19 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:19 rafthttp: failed to dial klm987 on stream Message (dial tcp 10.x.x.x:7001: i/o timeout)
Aug 19 16:39:19 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:19 rafthttp: failed to dial klm987 on stream MsgApp v2 (dial tcp 10.x.x.x:7001: i/o timeout)
Aug 19 16:39:20 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:20 rafthttp: failed to write klm987 on pipeline (dial tcp 10.x.x.x:2380: i/o timeout)
Aug 19 16:39:20 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:20 rafthttp: failed to dial klm987 on stream MsgApp v2 (dial tcp 10.x.x.x:2380: no route to host)
Aug 19 16:39:20 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:20 rafthttp: failed to dial klm987 on stream Message (dial tcp 10.x.x.x:2380: no route to host)
Aug 19 16:39:20 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:20 rafthttp: failed to write klm987 on pipeline (dial tcp 10.x.x.x:2380: no route to host)
Aug 19 16:39:21 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:21 rafthttp: failed to write klm987 on pipeline (dial tcp 10.x.x.x:7001: i/o timeout)
Aug 19 16:39:21 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:21 rafthttp: failed to dial klm987 on stream MsgApp v2 (dial tcp 10.x.x.x:7001: i/o timeout)
Aug 19 16:39:21 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:21 rafthttp: failed to dial klm987 on stream Message (dial tcp 10.x.x.x:7001: i/o timeout)
Aug 19 16:39:22 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:22 rafthttp: failed to write klm987 on pipeline (dial tcp 10.x.x.x:2380: i/o timeout)
Aug 19 16:39:22 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:22 rafthttp: failed to dial klm987 on stream MsgApp v2 (dial tcp 10.x.x.x:2380: i/o timeout)
Aug 19 16:39:22 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:22 rafthttp: failed to dial klm987 on stream Message (dial tcp 10.x.x.x:2380: i/o timeout)
Aug 19 16:39:23 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:23 etcdserver: failed to reach the peerURL(http://10.x.x.x:2380) of member klm987 (Get http://10.x.x.x:2380/version: net/http: request canceled while waiting for connection)
Aug 19 16:39:23 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:23 rafthttp: failed to write klm987 on pipeline (dial tcp 10.x.x.x:7001: no route to host)
Aug 19 16:39:23 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:23 etcdserver: failed to reach the peerURL(http://10.x.x.x:7001) of member klm987 (Get http://10.x.x.x:7001/version: dial tcp 10.x.x.x:7001: no route to host)
Aug 19 16:39:23 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:23 etcdserver: cannot get the version of member klm987 (Get http://10.x.x.x:7001/version: dial tcp 10.x.x.x:7001: no route to host)
Aug 19 16:39:23 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:23 rafthttp: failed to dial klm987 on stream Message (dial tcp 10.x.x.x:7001: no route to host)
Aug 19 16:39:23 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:23 rafthttp: failed to dial klm987 on stream MsgApp v2 (dial tcp 10.x.x.x:7001: no route to host)
Aug 19 16:39:24 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:24 rafthttp: failed to write klm987 on pipeline (dial tcp 10.x.x.x:2380: i/o timeout)
Aug 19 16:39:24 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:24 rafthttp: failed to dial klm987 on stream Message (dial tcp 10.x.x.x:2380: i/o timeout)
Aug 19 16:39:24 deis-v1.9.0-b90c2fbb-1 docker[675]: 2015/08/19 16:39:24 rafthttp: failed to dial klm987 on stream MsgApp v2 (dial tcp 10.x.x.x:2380: i/o timeout)
-- Logs begin at Wed 2015-08-19 10:34:41 UTC, end at Wed 2015-08-19 16:39:24 UTC. --
Aug 19 16:35:37 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO engine.go:79: Engine leader is hij78966d740379e6d8a0414fae730
Aug 19 16:36:52 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:36:52 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:36:52 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:36:53 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:36:53 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:36:53 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:36:53 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:36:53 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:36:53 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:36:53 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:36:53 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:36:53 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:36:53 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:36:53 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:36:53 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:36:53 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:36:53 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:36:53 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:36:53 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:36:54 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:36:54 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:36:55 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:36:56 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:36:56 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:36:56 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:36:57 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:36:57 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:36:58 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:37:37 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:37:37 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:37:38 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:37:38 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:37:38 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:37:38 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:37:38 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:37:38 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:37:38 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:37:38 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:37:38 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:37:38 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:37:38 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:37:38 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:37:38 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:37:38 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:37:38 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:37:38 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:37:38 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:37:40 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled
Aug 19 16:37:40 deis-v1.9.0-b90c2fbb-1 fleetd[913]: INFO client.go:292: Failed getting response from http://localhost:4001/: cancelled

--- DEIS STATUS ---
UNIT				MACHINE				LOAD	ACTIVE		SUB
deis-builder.service		abc123.../10.0.0.z	loaded	active		running
deis-controller.service		abc123.../10.0.0.z	loaded	active		running
deis-database.service		abc123.../10.0.0.z	loaded	active		running
deis-logger.service		efg456.../10.0.0.y	loaded	active		running
deis-logspout.service		efg456.../10.0.0.y	loaded	active		running
deis-logspout.service		abc123.../10.0.0.z	loaded	active		running
deis-logspout.service		hij789.../10.0.0._	loaded	active		running
deis-publisher.service		efg456.../10.0.0.y	loaded	active		running
deis-publisher.service		abc123.../10.0.0.z	loaded	active		running
deis-publisher.service		hij789.../10.0.0._	loaded	active		running
deis-registry@1.service		efg456.../10.0.0.y	loaded	active		running
deis-router@1.service		hij789.../10.0.0._	loaded	active		running
deis-router@2.service		efg456.../10.0.0.y	loaded	active		running
deis-router@3.service		abc123.../10.0.0.z	loaded	active		running
deis-store-daemon.service	efg456.../10.0.0.y	loaded	active		running
deis-store-daemon.service	abc123.../10.0.0.z	loaded	active		running
deis-store-daemon.service	hij789.../10.0.0._	loaded	active		running
deis-store-gateway@1.service	efg456.../10.0.0.y	loaded	active		running
deis-store-metadata.service	efg456.../10.0.0.y	loaded	inactive	dead
deis-store-metadata.service	abc123.../10.0.0.z	loaded	active		running
deis-store-metadata.service	hij789.../10.0.0._	loaded	active		running
deis-store-monitor.service	efg456.../10.0.0.y	loaded	active		running
deis-store-monitor.service	abc123.../10.0.0.z	loaded	active		running
deis-store-monitor.service	hij789.../10.0.0._	loaded	active		running
deis-store-volume.service	efg456.../10.0.0.y	loaded	active		running
deis-store-volume.service	abc123.../10.0.0.z	loaded	active		running
deis-store-volume.service	hij789.../10.0.0._	loaded	active		running
/deis/migrations
/deis/migrations/data
/deis/migrations/data/0001
/deis/migrations/data/0002
/deis/registry
/deis/registry/bucketName
/deis/registry/masterLock
/deis/registry/host
/deis/registry/port
/deis/registry/hosts
/deis/registry/hosts/10.0.0.y
/deis/registry/hosts/10.0.0.y/host
/deis/registry/hosts/10.0.0.y/port
/deis/registry/hosts/10.0.0.z
/deis/registry/hosts/10.0.0._
/deis/registry/protocol
/deis/config
/deis/config/production
/deis/config/production/redis_url
/deis/config/production/mailer_host
/deis/config/production/secret_token
/deis/config/production/asset_host
/deis/config/production/database_url
/deis/config/production/thread_pool
/deis/config/production/rails_env
/deis/config/production/s3_secret_access_key
/deis/config/production/s3_access_key_id
/deis/config/production/s3_bucket
/deis/config/production/new_relic_license_key
/deis/config/production/new_relic_app_name
/deis/config/production/workers
/deis/config/production/rack_env
/deis/config/production/buildpack_url
/deis/config/production/api_subdomain
/deis/config/production/frontend_subdomain
/deis/database
/deis/database/adminPass
/deis/database/user
/deis/database/initId
/deis/database/host
/deis/database/engine
/deis/database/adminUser
/deis/database/password
/deis/database/name
/deis/database/bucketName
/deis/database/port
/deis/router
/deis/router/hsts
/deis/router/gzip
/deis/router/proxyProtocol
/deis/router/hosts
/deis/router/hosts/10.0.0.z
/deis/router/hosts/10.0.0.y
/deis/router/hosts/10.0.0._
/deis/platform
/deis/platform/domain
/deis/platform/sshPrivateKey
/deis/platform/version
/deis/logs
/deis/logs/port
/deis/logs/drain
/deis/logs/host
/deis/cache
/deis/controller
/deis/controller/registrationMode
/deis/controller/port
/deis/controller/host
/deis/controller/protocol
/deis/controller/builderKey
/deis/controller/unitHostname
/deis/controller/subdomain
/deis/controller/secretKey
/deis/controller/webEnabled
/deis/services
/deis/services/production
/deis/domains
/deis/domains/new.mydomain.com
/deis/domains/new-api.mydomain.com
/deis/domains/www.mydomain.com
/deis/domains/api.mydomain.com
/deis/certs
/deis/store
/deis/store/fsid
/deis/store/monKeyring
/deis/store/gatewayKeyring
/deis/store/gateway
/deis/store/gateway/secretKey
/deis/store/gateway/masterLock
/deis/store/gateway/host
/deis/store/gateway/port
/deis/store/gateway/hosts
/deis/store/gateway/hosts/10.0.0.y
/deis/store/gateway/hosts/10.0.0.y/host
/deis/store/gateway/hosts/10.0.0.y/port
/deis/store/gateway/hosts/10.0.0.z
/deis/store/gateway/hosts/10.0.0.z/host
/deis/store/gateway/hosts/10.0.0.z/port
/deis/store/gateway/hosts/10.0.0._
/deis/store/gateway/hosts/10.0.0._/host
/deis/store/gateway/hosts/10.0.0._/port
/deis/store/gateway/accessKey
/deis/store/size
/deis/store/minSize
/deis/store/monSetupLock
/deis/store/delayStart
/deis/store/adminKeyring
/deis/store/osds
/deis/store/osds/10.0.0.y
/deis/store/osds/10.0.0._
/deis/store/osds/10.0.0.z
/deis/store/filesystemSetupLock
/deis/store/hosts
/deis/store/hosts/10.0.0._
/deis/store/hosts/10.0.0.y
/deis/store/hosts/10.0.0.z
/deis/store/pgNum
/deis/store/maxPGsPerOSDWarning
/deis/store/monSetupComplete
/deis/store/filesystemSetupComplete
/deis/builder
/deis/builder/users
/deis/builder/users/admin
/deis/builder/users/admin/b0:......
/deis/builder/sshHostecdsaKey
/deis/builder/sshHosted25519Key
/deis/builder/sshHosted25519PubKey
/deis/builder/sshHostKey
/deis/builder/sshHostdsaKey
/deis/builder/sshHostdsaPubKey
/deis/builder/sshHostecdsaPubKey
/deis/builder/sshHostrsaKey
/deis/builder/sshHostrsaPubKey
/deis/builder/sshHostPubKey
/deis/builder/host
/deis/builder/port
/deis/scheduler
```

**Comment by bacongobbler** [comment]
huh, that's an interesting error. Would you be willing to try the new [Go client](https://github.com/deis/deis/tree/master/client-go)? We're looking to [cut over this release](https://github.com/deis/deis/pull/4261) and it might unblock you. Let me know and I can build you a v1.9.0 client :)

**Comment by bf4** [comment]
@bacongobbler I can build it, I think. Looks like I need to install the docker client, which I think mean `brew install docker`? but not boot2docker?

**Comment by bacongobbler** [comment]
All you need is Go 1.4 and Godep :) https://github.com/deis/deis/blob/master/client-go/Makefile#L14

**Comment by bf4** [comment]
Hmm, I'm on go 1.4.2 on master and `make build` outputs (same with go install)...

```plain
CGO_ENABLED=0 godep go build -a -installsuffix cgo -ldflags '-s' -o deis .
deis.go:10:2: cannot find package "github.com/deis/deis/client-go/parser" in any of:
        /usr/local/Cellar/go/1.4.2/libexec/src/github.com/deis/deis/client-go/parser (from $GOROOT)
        /Users/bfleischer/projects/deis/Godeps/_workspace/src/github.com/deis/deis/client-go/parser (from $GOPATH)
        /Users/bfleischer/gocode/src/github.com/deis/deis/client-go/parser
deis.go:11:2: cannot find package "github.com/deis/deis/version" in any of:
        /usr/local/Cellar/go/1.4.2/libexec/src/github.com/deis/deis/version (from $GOROOT)
        /Users/bfleischer/projects/deis/Godeps/_workspace/src/github.com/deis/deis/version (from $GOPATH)
        /Users/bfleischer/gocode/src/github.com/deis/deis/version
godep: go exit status 1
make: *** [build] Error 1
```

`make` on its own says

```plain
~/projects/deis/client-go
$ make
Missing `docker` client which is required for development
make: *** [check-docker] Error 2
```

**Comment by bacongobbler** [comment]
You'll have to move deis/deis to `$GOPATH/src/github.com/deis/deis` or else `godep` wont be able to find anything under the "github.com/deis/deis" package.

**Comment by bf4** [comment]
Ok, got it to build but haven't played with it yet.  In the meantime, pip version is working fine.

Here's my steps from start to finish :)

```bash
unset GOROOT
export GOPATH="$HOME/gocode"
echo $GOPATH
go env

go get github.com/tools/godep

mkdir -p  $GOPATH/src/github.com/deis
cd $GOPATH/src/github.com/deis
git clone deis/deis
export DEIS=$GOPATH/src/github.com/deis/deis

cd $DEIS
git remote add upstream https://github.com/deis/deis.git

cd $DEIS/deisctl
  make build
 # Running make install will install the deisctl command in $GOPATH/bin/deisctl
 ./deisctl -h

cd $DEIS/client-go/
  make build
  # Running make install will install the deis command in $GOPATH/bin/deis
 ./deis -h
 ./deis --version
```

**Comment by mboersma** [comment]
The python client is deprecated and will be removed with the v1.10.0 release. If you see similar issues with the new go CLI, please open a new issue.
